internal/server/v1/propertyvalues/golden/bulk_property_values_in/containedIn2.json

```diff
@@ -22457,7 +22457,7 @@
             "City"
           ],
           "dcid": "geoId/0628021",
-          "provenanceId": "dc/5n63hr1"
+          "provenanceId": "dc/sm3m2w3"
```

Comment:
This is not expected after the fix in cl/485779075.  But the previous IPCC version was built *before* the CL went in, and there could be something with the merge leading to this.  So this might be ok, but worth keeping an eye on future diffs.